docs/tables/aws_ecs_container_instance.md

```diff
@@ -0,0 +1,62 @@
+# Table: aws_ecs_container_instance
+
+An Amazon ECS container instance is a EC2 instance that is running the Amazon ECS agent and has been registered with an ECS cluster.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  container_instance_arn,
+  instance_id,
+  status,
+  status_reason,
+  running_tasks_count,
+  pending_tasks_count
+from
+  aws_ecs_container_instance;
+```
+
+
+### List container instances that have failed registration
```

Comment:
Are these leftover from potentially copying the `docs/tables/aws_ecs_cluster.md` content since they reference the `aws_ecs_cluster` table, or are these examples meant to be in this doc?

Reply:
Yeah, these should be updated.